src/program/lwaftr/tests/soaktest/core-soaktest.sh

```diff
@@ -13,10 +15,10 @@ function quit_with_msg {
 
 function soaktest {
     conf="$1"; in_v4="$2"; in_v6="$3"
-    $SNABB_LWAFTR soaktest "$conf" "$in_v4" "$in_v6" ||
-        quit_with_msg $? "Test failed: $SNABB_LWAFTR soaktest $@"
-    $SNABB_LWAFTR soaktest --on-a-stick "$conf" "$in_v4" "$in_v6" ||
-        quit_with_msg $? "Test failed: $SNABB_LWAFTR soaktest --on-a-stick $@"
+    $SNABB_LWAFTR soaktest -D $DURATION "$conf" "$in_v4" "$in_v6" ||
+        quit_with_msg $? "$SNABB_LWAFTR soaktest -D $DURATION $conf $in_v4 $in_v6"
+    $SNABB_LWAFTR soaktest -D $DURATION --on-a-stick "$conf" "$in_v4" "$in_v6" ||
+        quit_with_msg $? "$SNABB_LWAFTR soaktest -D $DURATION --on-a-stick $conf $in_v4 $in_v6"
```

Comment:
Yes, actually it's a fix. With the former code that used $@, more parameters than needed where shown so when trying to run the test case it failed cause `lwaftr soaktest` expects 3 parameters only: conf, in_ipv4 and ip_ipv6. 

Before the failing command shown 7 parameters because this function actually receives that many parameters. Function `read_test_data`, that is used read test data in groups of 7 parameters, necessary for running end-to-end test cases. The relevant line is this one:

https://github.com/Igalia/snabb/blob/lwaftr/src/program/lwaftr/tests/soaktest/core-soaktest.sh#L29